### DISCUSSION TITLE
What is HDR Plus?

### DISCUSSION THREAD

**Discussion by Gammler** [VirtualDrivers/Virtual-Display-Driver#182] [Q&A] [answered]
I just installed the new Virtual Display Driver Beta 24.10.27.

What is HDR Plus. There is not explaination to find.

I also find this term "HDR Plus" a bad choice. Because there are official standards HDR10 and HDR10+ and this confuses me.
But HDR+ or HDR Plus is nothing official to my knowledge

**Comment by bud3699** [comment, marked as answer]
If hdr+ doesnt exist its safe to assume it means hdr10+, same way with saying we have hdr support, we dont say we have hdr10 support. Although I appreciate your input, this will not be changed
